utils/build_video_reid_dataset.py

```diff
@@ -0,0 +1,190 @@
+# DATASET_OUTPUT_LOCATION = '/mnt/raid1/home/bar_cohen/OUR_DATASETS/DukeMTMC-VideoReID' # keep this, it is for video location
+DATASET_OUTPUT_LOCATION = '/home/bar_cohen/KinderGuardian/fast-reid/datasets/same_day_0308' # this is for image
+
+
+def im_name_in_mars(crop: Crop, track_counter, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}C{crop.cam_id}T{track_counter:04d}F{crop_id:03d}.png'
+
+def im_name_in_duke(crop: Crop, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}_C{crop.cam_id}_F{crop_id + 1:04d}_X{crop.frame_num:05d}.png'
+
+def im_name_in_img_duke(crop:Crop, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}_c{crop.cam_id}_f{crop_id:07d}.jpg'
+
+def convert_to_img_reid_duke_naming(dataset_path:str, query_days, same_day=False):
+    # entering 'same_day' will result in a split only to query, train and test only from this day
+
+    df = pd.DataFrame(columns=['file_name', 'vid_name', 'track_id'])
+    warnings.warn("Query will be taken from same day as test")
+    video_names = [vid.vid_name for vid in get_entries(filters=(),group=Crop.vid_name)]
+    crop_counter  = 0
+    for vid_name in video_names:
+        if vid_name[0:8] == same_day or vid_name in query_days:
+            set_folder = 'query'
+        else:
+            if same_day:
+                continue
+            set_folder = 'bounding_box_train'
+        print('running', vid_name)
+        tracks = [track.track_id for track in get_entries(filters=({Crop.vid_name == vid_name}), group=Crop.track_id)]
+        for track in tracks:
+            track_crops = get_entries(filters=(Crop.vid_name == vid_name,
+                                               Crop.track_id == track,
+                                               Crop.reviewed_one == True,
+                                               Crop.invalid == False)).all()
+            if same_day and random.random() <= 0.8:
+                set_folder = 'bounding_box_train'
+
+            for crop in track_crops:
+                output_name = im_name_in_img_duke(crop, crop_counter)
+                orig_crop_path = os.path.join(BASE_CROPS_LOCATION, crop.vid_name, crop.im_name)
+                dataset_crop_path = os.path.join(dataset_path, set_folder, output_name)
+                os.makedirs(os.path.join(dataset_path, set_folder), exist_ok=True)
+                shutil.copy(orig_crop_path, dataset_crop_path)
+                if set_folder == 'bounding_box_train':  # add to gallery as well
+                    dataset_crop_path = os.path.join(dataset_path, 'bounding_box_test', output_name)
+                    os.makedirs(os.path.join(dataset_path, 'bounding_box_test'), exist_ok=True)
+                    shutil.copy(orig_crop_path, dataset_crop_path)
+                crop_counter += 1
+                df = add_file_data_base(df, output_name, crop)
+    df.to_csv(os.path.join(DATASET_OUTPUT_LOCATION, 'img_to_info.csv'))
+
+
+def convert_to_mars_naming(dataset_path: str):
```

Comment:
Can we remove this function or are we planning to use the MARS notation someday?

Reply:
I say keep it, we might want to check on MARS later on